sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -396,23 +440,43 @@ private[spark] object HiveUtils extends Logging {
         config = configurations,
         barrierPrefixes = hiveMetastoreBarrierPrefixes,
         sharedPrefixes = hiveMetastoreSharedPrefixes)
+    } else if (hiveMetastoreJars == "path") {
+      // Convert to files and expand any directories.
+      val jars =
+        HiveUtils.hiveMetastoreJarsPath(sqlConf)
+          .flatMap {
+            case path if path.contains("\\") && Utils.isWindows =>
```

Comment:
I don't know windows very well. cc @HyukjinKwon

Reply:
> I don't know windows very well. cc @HyukjinKwon

see https://github.com/apache/spark/blob/77a8efbc05cb4ecc40dd050c363429e71a9f23c1/core/src/main/scala/org/apache/spark/SparkContext.scala#L1902

Reply:
Yeah, this is fine.